jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/CloudWatchLogWindow.kt

```diff
@@ -89,8 +89,16 @@ class CloudWatchLogWindow(private val project: Project) : CoroutineScope by Appl
         }
     }
 
+    fun closeLogGroup(logGroup: String) = launch {
+        toolWindow.findPrefix(logGroup).forEach {
```

Comment:
We should find a better route, this will close FooBar if you delete Foo, or mutli-account with FooBar open

Reply:
I know this happens but I didn't want to fix it because it's a rare case, will do a quick fix. We should probably write a toolwindowmangerV2 that makes stuff like this easier as we have added a lot more expectations on top of this vs just opening a log or cloudformation stack.

Reply:
Lots of V2 stuff we need to write....